.github/workflows/check_in_artifact.yml

```diff
@@ -117,15 +117,15 @@ jobs:
           PR_TITLE: ${{ inputs.pull_request_title }}
           PR_BODY: ${{ inputs.pull_request_body }}
         run: |
-          EXISTING_CLOSED_PR="$(gh pr list --state closed --base master --head $HEAD_BRANCH_NAME --json url --jq '.[].url')"
+          EXISTING_CLOSED_PR="$(gh pr list --state closed --base master --head $HEAD_BRANCH_NAME --json url --jq '.[0].url')"
```

Comment:
Had to index the first entry because if someone closed multiple PRs using the same branch, then `gh pr list` will return multiple URLs, and we only want the most recent one.

Reply:
I tried it locally and indexing just gave an empty string. And the exit code is 0 so it doesn't cause the workflow to fail. The later conditional checks for empty strings anyway, so it works out perfectly 😄